text.cabal

```diff
@@ -147,7 +147,8 @@ library
     base       >= 4.2 && < 5,
     binary,
     deepseq    >= 1.1.0.0,
-    ghc-prim   >= 0.2
+    ghc-prim   >= 0.2,
+    template-haskell
```

Comment:
Hi, just curious, was any thought put into adding this extra dependency? Now, packages which use `text` will now need to build/link `template-haskell`. Is this not that a big enough issue for most people?

Reply:
Note that `template-haskell` is considered a non-reinstallable package; i.e. it's only built once when GHC itself is build. Also note this doesn't require a GHC with TH engine either (i.e. `text` still works perfectly well with a GHC that cannot run TH). But why do you consider the need to link against `template-haskell` a problem?

Reply:
If a Haskell executable statically links their dependencies, wouldn't it become non-trivially bigger having to statically link `template-haskell`, even if they aren't using the `Lift` instance for `Text`?